src/features/session_replay/shared/recorder.js

```diff
@@ -166,7 +166,12 @@ export class Recorder {
 
   /** force the recording lib to take a full DOM snapshot.  This needs to occur in certain cases, like visibility changes */
   takeFullSnapshot () {
-    recorder.takeFullSnapshot()
+    try {
+      if (!this.recording) return
+      recorder.takeFullSnapshot()
```

Comment:
Should we abort the network call as well?

Reply:
By the time we get to this the network call should have resolved so we would need to abort elsewhere and TBH i dont think its worth the extra complexity to manage